eclipse-scout-core/src/tile/TileGrid.ts

```diff
@@ -252,71 +253,83 @@ export class TileGrid extends Widget implements TileGridModel {
     }
   }
 
-  insertTile(tile: ObjectOrChildModel<Tile>) {
+  insertTile(tile: ObjectOrChildModel<TTile>) {
     this.insertTiles([tile]);
   }
 
-  insertTiles(tilesToInsert: ObjectOrChildModel<Tile> | ObjectOrChildModel<Tile>[], appendPlaceholders?: boolean) {
+  insertTiles(tilesToInsert: ObjectOrChildModel<TTile> | ObjectOrChildModel<TTile>[]) {
+    this.insertTilesInternal(tilesToInsert);
+  }
+
+  protected insertTilesInternal(tilesToInsert: ObjectOrChildModel<TTile | PlaceholderTile> | ObjectOrChildModel<TTile | PlaceholderTile>[], appendPlaceholders?: boolean) {
     tilesToInsert = arrays.ensure(tilesToInsert);
     if (tilesToInsert.length === 0) {
       return;
     }
-    let tiles = this.tiles as ObjectOrChildModel<Tile>[];
-    this.setTiles(tiles.concat(tilesToInsert), appendPlaceholders);
+    let tiles = this._tiles as ObjectOrChildModel<TTile | PlaceholderTile>[];
+    this.setTilesInternal(tiles.concat(tilesToInsert), appendPlaceholders);
   }
 
-  deleteTile(tile: Tile) {
+  deleteTile(tile: TTile) {
     this.deleteTiles([tile]);
   }
 
-  deleteTiles(tilesToDelete: Tile | Tile[], appendPlaceholders?: boolean) {
+  deleteTiles(tilesToDelete: TTile | (TTile)[]) {
+    this.deleteTilesInternal(tilesToDelete);
+  }
+
+  protected deleteTilesInternal(tilesToDelete: (TTile | PlaceholderTile) | (TTile | PlaceholderTile)[], appendPlaceholders?: boolean) {
```

Comment:
Is the first parenthesis necessary?
"TTile | PlaceholderTile | (TTile | PlaceholderTile)[]" looks a bit cleaner